pkg/istio-agent/agent.go

```diff
@@ -88,6 +88,9 @@ const (
 	MetadataClientCertKey   = "ISTIO_META_TLS_CLIENT_KEY"
 	MetadataClientCertChain = "ISTIO_META_TLS_CLIENT_CERT_CHAIN"
 	MetadataClientRootCert  = "ISTIO_META_TLS_CLIENT_ROOT_CERT"
+	CertSignerLabel         = "cert.istio.io/signer"
```

Comment:
can you add comments? especially what the format is?

Also this should be in istio/api, under the labels/ package.

Reply:
Thanks for this pointer! Should we mark this as `EXPERIMENTAL` ?